lib/src/execution/windows/execution.cc

```diff
@@ -358,6 +358,24 @@ namespace facter { namespace execution {
         option_set<execution_options> const& options,
         uint32_t timeout)
     {
+        // Since we use a job object in the windows world, we want to
+        // be sure we're not in a job object, or at least able to
+        // break our processes out if we are in one.
+        BOOL in_job;
+        bool use_job_object = true;
+        if (!IsProcessInJob(GetCurrentProcess(), nullptr, &in_job)) {
+            throw execution_exception("could not determine if facter is in a object");
```

Comment:
Did the message intend to say `could not determine if facter is is running in a job` or some such?  Do we know when `IsProcessInJob` can fail?